examples/make_flow/make_realnvp_normflow.py

```diff
@@ -58,3 +93,30 @@ def make_normflow_model(
     if act_norm:
         model.sample(500)  # ensure we call sample to initialise the ActNorm layers
     return model
+
+
+def make_normflow_snf_model(
```

Comment:
@lollcat you should be able to use this function as a replacement of `make_normflow_model` to create and SNF. The default parameters are similar to what has been used in the original paper.